origins_social_sharing/templates/origins-social-sharing.html.twig

```diff
@@ -14,28 +14,31 @@
   <span class="social-sharing__label">{{ "Share this page"|t }}</span>
   <span class="social-sharing__buttons">
       <a class="facebook-share share-link no-ext-icon" target="_blank" href="{{ fb_link }}" rel="nofollow noopener">
-        <span class="visually-hidden">Share on Facebook (external link opens in a new window / tab)</span>
         <span class="share-link__button">
           <svg role="img" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
+            <title>Share on Facebook (external link opens in a new window / tab)</title>
```

Comment:
Need to remove the extra "(external link opens new window / tab)" in the SVG titles as same string is output in visually hidden span after the svg